File: Card.java
public class Card {

    private final String question;
    private final String answer;
    private final boolean correct;

    public Card(String question, String answer, boolean correct) {
        this.question = question;
        this.answer = answer;
        this.correct = correct;
    }

    public String getQuestion() {
        return question;
    }

    public String getAnswer() {
        return answer;
    }

    public boolean isCorrect() {
        return correct;
    }
}

File: Card.ts
export class Card {
    private readonly question: string;
    private readonly answer: string;
    private readonly correct: boolean;

    constructor(question: string, answer: string, correct: boolean) {
        this.question = question;
        this.answer = answer;
        this.correct = correct;
    }

    getQuestion(): string {
        return this.question;
    }

    getAnswer(): string {
        return this.answer;
    }

    isCorrect(): boolean {
        return this.correct;
    }
}